runtime/tr.source/trj9/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -3797,6 +3797,12 @@ TR::Register *J9::Power::TreeEvaluator::VMinstanceOfEvaluator2(TR::Node *node, T
             if (comp->getOption(TR_TraceCG)) traceMsg(comp, "%s: Emitting ArrayOfJavaLangObjectTest\n", node->getOpCode().getName());
             genInstanceOfOrCheckCastObjectArrayTest(node, cr0Reg, objectClassReg, nextSequenceLabel, srm, cg);
             break;
+         case DynamicCacheObjectClassTest:
```

Comment:
FYI @ymanton @gita-omr  I have added this two cases in the instanceOf evaluator of P which just prints message on traceLog that test is not implemented. It does not affect any code generation on P.

Reply:
Would adding a `TR_ASSERT_FATAL` not be better here instead of a trace message that can easily be ignored?

Reply:
Common heuristics is going to add these tests in the list of tests to be generated and it will be common to reach this path as it will be added test if cast class is runtime variable or interface but it is not necessary to generate test. I am not sure if `TR_ASSERT_FATAL` is right thing to do here it is most likely that we are going to execute this path. 
Depending on the cost of generating this tests on P, they can chose not to implement it

Reply:
I see. I incorrectly assumed the test would not be generated on Power to begin with.